### ISSUE TITLE
AimLogger error on new Pytorch-Lightning Release

### ISSUE THREAD

**Issue by haritsahm** [aimhubio/aim#1619]
## 🐛 Bug

Removed `_convert_params` function in new pytorch-lightning release (1.6.0). 

### To reproduce

```
from aim.pytorch_lightning import AimLogger

trainer = pl.Trainer(logger=AimLogger(experiment='experiment_name'))
trainer.logger.log_hyperparams(hparams)
```
### Expected behavior

- Successfully log the hyperparameters with no errors

### Environment

- Aim Version (3.8.0)
- Python version (3.8.12)
- pip version (21.2.4)
- OS (e.g., Linux)
- Pytorch-lightning (1.6.0)
## Error output
```
Error executing job with overrides: ['experiment=dma_net', 'debug=default', 'log_dir=debug', 'data_dir=/media/haritsahm/DataStorage/dataset/cityscapes/cityscape_fo_segmentation/', 'logger=aim']
Traceback (most recent call last):
  File "train.py", line 22, in main
    return train(config)
  File "/home/haritsahm/Documents/projects/deeplearning/pytorch-DMANet/src/training_pipeline.py", line 74, in train
    utils.log_hyperparameters(
  File "/opt/conda/lib/python3.8/site-packages/pytorch_lightning/utilities/rank_zero.py", line 32, in wrapped_fn
    return fn(*args, **kwargs)
  File "/home/haritsahm/Documents/projects/deeplearning/pytorch-DMANet/src/utils/__init__.py", line 143, in log_hyperparameters
    trainer.logger.log_hyperparams(hparams)
  File "/opt/conda/lib/python3.8/site-packages/pytorch_lightning/utilities/rank_zero.py", line 32, in wrapped_fn
    return fn(*args, **kwargs)
  File "/opt/conda/lib/python3.8/site-packages/aim/sdk/adapters/pytorch_lightning.py", line 70, in log_hyperparams
    params = self._convert_params(params)
AttributeError: 'AimLogger' object has no attribute '_convert_params'

Set the environment variable HYDRA_FULL_ERROR=1 for a complete stack trace.
root@0e330cd4b21f:/home/haritsahm/Documents/projects/deeplearning/pytorch-DMANet# 

```

**Comment by mihran113** [comment]
hey @haritsahm. Thanks for the report. I've already opened a PR for the fix. It'll be shipped with the `aim v3.9.0` and is planned for someday in the start of the next week. If it's a blocker, you can use the upcoming nightly release which will include the fix in it, as it's planned to be merged today. It'll be available after Apr 9, 00:00 GMT and can be installed by command `pip install --pre aim`

**Comment by djwessel** [comment]
@mihran113 Hey, not sure if I should be opening this in a separate ticket or not, but since we are talking about compatibility with the latest PL version, as of PL v1.5.0, the `LightningLoggerBase.close` method has been depreciated in favor of `LightningLoggerBase.finalize`.

https://pytorch-lightning.readthedocs.io/en/stable/api/pytorch_lightning.loggers.base.html#pytorch_lightning.loggers.base.LightningLoggerBase.close

**Comment by mihran113** [comment]
Hey @djwessel. Thanks a lot for the report. Will have a look into it.
Would be super helpful, if you can open a separate issue for it as this one's already fixed and waiting to be shipped. 🙌

**Comment by djwessel** [comment]
Opened #1660 for handling the deprecation.

**Comment by mpds** [comment]
Hi, I've installed the nightly release (using the `pip install --pre` flag), but the error persists. 

## To reproduce
```python
trainer = pl.Trainer.from_argparse_args(
    args, logger=AimLogger(experiment="aim_test")
)
```

## Environment
- pytorch-lightning==1.6.1
- Python==3.9.10
- aim==3.9.0a14
- aimrecords==0.0.7
- aimrocks==0.1.3a14

## Error output
```bash
Traceback (most recent call last):
  File "/home/mpds/miniconda3/envs/env/lib/python3.9/runpy.py", line 197, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "/home/mpds/miniconda3/envs/env/lib/python3.9/runpy.py", line 87, in _run_code
    exec(code, run_globals)
  File "/home/mpds/python/ml/scripts/train.py", line 90, in <module>
    main(parser.parse_args())
  File "/home/mpds/python/ml/scripts/train.py", line 62, in main
    trainer.fit(
  File "/home/mpds/miniconda3/envs/env/lib/python3.9/site-packages/pytorch_lightning/trainer/trainer.py", line 768, in fit
    self._call_and_handle_interrupt(
  File "/home/mpds/miniconda3/envs/env/lib/python3.9/site-packages/pytorch_lightning/trainer/trainer.py", line 721, in _call_and_handle_interrupt
    return trainer_fn(*args, **kwargs)
  File "/home/mpds/miniconda3/envs/env/lib/python3.9/site-packages/pytorch_lightning/trainer/trainer.py", line 809, in _fit_impl
    results = self._run(model, ckpt_path=self.ckpt_path)
  File "/home/mpds/miniconda3/envs/env/lib/python3.9/site-packages/pytorch_lightning/trainer/trainer.py", line 1222, in _run
    self._log_hyperparams()
  File "/home/mpds/miniconda3/envs/env/lib/python3.9/site-packages/pytorch_lightning/trainer/trainer.py", line 1290, in _log_hyperparams
    logger.log_hyperparams(hparams_initial)
  File "/home/mpds/miniconda3/envs/env/lib/python3.9/site-packages/pytorch_lightning/utilities/rank_zero.py", line 32, in wrapped_fn
    return fn(*args, **kwargs)
  File "/home/mpds/miniconda3/envs/env/lib/python3.9/site-packages/aim/sdk/adapters/pytorch_lightning.py", line 70, in log_hyperparams
    params = self._convert_params(params)
AttributeError: 'AimLogger' object has no attribute '_convert_params'

```

**Comment by mihran113** [comment]
hey @mpds! Sorry for inconvenience, but that's actually a different `dev` release that doesn't have the fix included in it. Can I ask you to try out this one and let me know if it works properly? 
```
pip install aim==3.9.0.dev20220417
```

**Comment by mpds** [comment]
Hi @mihran113, it worked! Thanks!

**Comment by mihran113** [comment]
Hey folks! Glad to inform that the fix was shipped with the new `aim v3.9` release, please have a try, and let me know if everything works as expected.
@mpds @haritsahm @djwessel
